src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp

```diff
@@ -1079,8 +1079,8 @@ class instance_trial_of_the_champion : public InstanceMapScript
                     {
                     if (Creature* boss = instance->GetCreature(NPC_ArgentChampionGUID))
                         boss->GetMotionMaster()->MovePoint(0, 746.881f, 635.263f, 411.7f);
-                        events.ScheduleEvent(EVENT_ARGENT_CHALLENGE_ATTACK, 3000);
-                        events.PopEvent();
+                    events.ScheduleEvent(EVENT_ARGENT_CHALLENGE_ATTACK, 3000);
+                    events.PopEvent();
```

Comment:
Note: this might be a bug. Here I'm just changing the indentation to make it clear that not all statements are part of the IF block. **I'm NOT changing the behaviour**.

Reply:
same here

Reply:
done